doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -72,31 +64,30 @@ Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
       # GRUB_HIDDEN_TIMEOUT=0
       GRUB_HIDDEN_TIMEOUT_QUIET=false
 
-#. Update grub::
+#. Update GRUB::
 
    $ sudo update-grub
 
 #. Reboot the NUC. Select the **ACRN hypervisor Hybrid Scenario** entry to boot
    the ACRN hypervisor on the NUC’s display. The GRUB loader will boot the
-   hypervisor, and the hypervisor will start VMs automatically.
+   hypervisor, and the hypervisor will start the VMs automatically.
 
 Hybrid Scenario Startup Checking
 ********************************
-#. Use these steps to verify the hypervisor is properly running:
+#. Use these steps to verify that the hypervisor is properly running:
 
-   a. Login ACRN hypervisor shell from serial console.
+   a. Log in to the ACRN hypervisor shell from the serial console.
```

Comment:
On the next line, change to ``Use the vm_list command to check the pre-launched ...``